src/browser/icons/add-circle.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor">
+	<g>
```

Comment:
I think tab indentation is inconsistent with the other SVGs

Reply:
Hmm they seem to re-appear on save when I remove them. I'll look into it in another PR